src/charm.py

```diff
@@ -26,6 +28,7 @@
 from literals import APP_NAME, APPLICATION_PORT
 from log import log_event_handler
 from postgresql import Database
+from redis import Redis
```

Comment:
As the number of relations increases, it might be worth creating a ```relations``` directory.

Reply:
Good idea, added :)